### PR TITLE
[GNA] Support of sub requests 

### PR DESCRIPTION
**Pull request by mryzhov**
### Details:
 - Excluded unused tuples
 - Added sub requests processing

### Tickets:
 - N/A

### CODE REVIEW

File: src/plugins/intel_gna/gna_data_types.hpp [hunk 1]
```diff
@@ -23,6 +23,13 @@ struct TranspositionInfo {
     size_t num_transpose_columns;
 };
 
+struct Gna2RequestConf {
+    uint32_t id = 0;
```
Comment (nkogteva) [inline src/plugins/intel_gna/gna_data_types.hpp:27]: why not `FAKE_REQUEST_CONFIG_ID` is used as default?

**Review by nkogteva — APPROVED** [review]

**Review by kbruniec — CHANGES_REQUESTED** [review]
It looks like this PR is adding dead code (unused structure) only.

**Comment by mryzhov** [comment]
> It looks like this PR is adding dead code (unused structure) only.
Yes, I'm rebasing the old PRs, and looks like those code has been submitted already

**Comment by mryzhov** [comment]
no need anymore because those changes were submitted as part of other PRs

**Pull request closed without merging**